cmd/kops/create_cluster.go

```diff
@@ -489,8 +488,8 @@ func RunCreateCluster(ctx context.Context, f *util.Factory, out io.Writer, c *Cr
 	if c.ContainerRuntime != "" {
 		cluster.Spec.ContainerRuntime = c.ContainerRuntime
 	}
-	if c.ContainerRuntime == "containerd" && components.UsesKubenet(cluster.Spec.Networking) {
-		return fmt.Errorf("--networking with CNI plugin is required for containerd")
+	if c.ContainerRuntime == "containerd" && c.Networking == "" {
```

Comment:
we default to networking = kubenet correct? if the default `--networking` value now works with containerd then we should just drop this "if" statement altogether rather than require a `--networking` flag.

Reply:
True. I wanted to force people to choose. Removed.